projects/ui/src/lib/components/po-widget/po-widget-base.component.ts

```diff
@@ -175,22 +175,22 @@ export abstract class PoWidgetBaseComponent {
   }
 
   /** Ação que será executada quando o usuário clicar sobre a área total do `po-widget`. */
-  @Output('p-click') click?: EventEmitter<MouseEvent> = new EventEmitter<MouseEvent>();
+  @Output('p-click') click: EventEmitter<MouseEvent> = new EventEmitter<MouseEvent>();
 
   /** Função que será disparada com o valor do `p-disabled` quando esta propriedade for alterada. */
-  @Output('p-on-disabled') onDisabled?: EventEmitter<any> = new EventEmitter<any>();
+  @Output('p-on-disabled') onDisabled: EventEmitter<any> = new EventEmitter<any>();
 
   /** Função que será chamada na primeira ação. */
-  @Output('p-primary-action') primaryAction?: EventEmitter<any> = new EventEmitter<any>();
+  @Output('p-primary-action') primaryAction: EventEmitter<any> = new EventEmitter<any>();
 
   /** Função que será chamada na segunda ação. */
-  @Output('p-secondary-action') secondaryAction?: EventEmitter<any> = new EventEmitter<any>();
+  @Output('p-secondary-action') secondaryAction: EventEmitter<any> = new EventEmitter<any>();
 
   /** Função chamada ao clicar no ícone de configuração */
```

Comment:
O `?` era utilizado para documentação também, com isso no Portal aparecia `(opcional)`.
Com essa atualização os eventos não apresentarão mais o `opcional`.

Mas tem também como fazer via documentação, veja abaixo:

```suggestion 
 /**
   * @optional
   *
   * @description
   * Função chamada ao clicar no ícone de configuração
   */
```

Reply:
Pronto!